frontend/src/pages/Admin/components/CsvImport.js

```diff
@@ -90,6 +96,13 @@ export default function CsvImport(
       return;
     }
 
+    // Verify correct encoding.
+    const encodingInfo = await languageEncoding(files[0]);
```

Comment:
Do we care to handle possible rejection here, or do we think it's too unlikely to bother with?

Reply:
What do you mean by that?